src/shared/install.c

```diff
@@ -1492,7 +1492,7 @@ static int install_info_symlink_alias(
                 rp = skip_root(paths, i->path);
 
                 q = create_symlink(rp ?: i->path, alias_path, force, changes, n_changes);
-                if (r == 0)
+                if (q < 0)
                         r = q;
```

Comment:
But the condition is always true, no? There is no code path where `r != 0`.

And `create_symlink()` returns 1 if a symlink was created or already exists. So `install_info_symlink_alias()` returns failure when it should not.